schemas/JoinedRoom.json

```diff
@@ -15,7 +15,9 @@
        {"const": "RoomDirectory", "description": "Room joined via the public rooms directory."},
        {"const": "SpaceHierarchy", "description": "Room joined via the space hierarchy view."},
        {"const": "Timeline", "description": "Room joined via a timeline pill or link in another room."},
-       {"const": "Notification", "description": "Room joined via a push/desktop notification."}
+       {"const": "Notification", "description": "Room joined via a push/desktop notification."},
+       {"const": "MobilePermalink", "description": "Room joined via link"},
+       {"const": "MobileExploreRooms", "description": "Room joined via space explore"}
```

Comment:
This one could be `SpaceHierarchy` which is already defined here and matches ViewRoom

Reply:
I wasn't sure what "space hierarchy" stands for. Is it just "Explore Rooms" screen or something else?

Reply:
> Room joined via the space hierarchy view.

That's the explore space view, hierarchy because in some implementations (Web) you can see subspaces and subspace-rooms etc